source/includes/_outliers.md

```diff
@@ -12,24 +12,24 @@ errors occurred. If no validation outliers have been detected, the `outliers:[]`
 ```json
 [
     {
-        "gresb_asset_id": 357239,
+        "gresb_asset_id": {asset_id},
         "country": "US",
         "state_province": "DC",
         "city": "Washington, DC",
         // ... trimmed for brevity ...
         "certifications": [   {
-            "id": 10467,
+            "id": {assigned_certification_id},
```

Comment:
simply `{certification_id}`?

Reply:
It is not visible in the above code but there are two IDs in the certification, "id" is assigned by GRESB when certification is added to the asset and "certification_id" identifies the certification scheme (e.g BREEAM). I wanted to differentiate between the two to avoid confusion.
![image](https://user-images.githubusercontent.com/100768664/208408718-40113d20-31db-4814-a409-e5510f07b789.png)
![image](https://user-images.githubusercontent.com/100768664/208409923-399154b8-d999-4556-9a45-d0c861a7528d.png)

Reply:
Since this is the terminology in the api-docs:

>The response includes a unique id, which is the identifier for the particular association record created.

It might be best to use `unique_id`